substrate-sdk-android/src/test/java/io/novasama/substrate_sdk_android/encrypt/SignatureVerifierTest.kt

```diff
@@ -73,6 +93,31 @@ class SignatureVerifierTest {
         assertEquals(shouldBeValid, isValid)
     }
 
+    private fun signAndVerifySr25519WithEncryptedKey(
```

Comment:
What does this test actually check? Since the code we change is related to derivation the test should be in the derivations test cases and should check that we do the derivation correctly:

1. Get the private key from some Vault account. Also get the account id. Use it as the input data
2. Derive the keypair
3. Check that the publicKey matches the account id from Vault. Also check that the secret key (private key + nonce) is the same as the one that was passed

For the seed derivation, do the same, but check just the public key